index.js

```diff
@@ -24,16 +27,26 @@ var Dropzone = React.createClass({
     activeStyle: React.PropTypes.object,
     className: React.PropTypes.string,
     activeClassName: React.PropTypes.string,
+    rejectClassName: React.PropTypes.string,
 
     disableClick: React.PropTypes.bool,
     multiple: React.PropTypes.bool,
+    accept: React.PropTypes.string,
+  },
+
+  allFilesAccepted: function(files) {
+    return files.every(file => accept(file, this.props.accept))
```

Comment:
Right, but not sure it will be transformed with current settings. Let me re-check and send a new PR.